packages/openapi-typescript/src/load.ts

```diff
@@ -173,7 +173,7 @@ export default async function load(schema: URL | Subschema | Readable, options:
   else if (typeof schema === "object") {
     options.schemas[schemaID] = {
       hint: "OpenAPI3",
-      schema: schema as any,
+      schema: "structuredClone" in globalThis ? structuredClone(schema as any) : schema, // TODO: `"structuredClone" in globalThis ?` only temporarily needed until Node 16 is EOL on 2023-09-11
```

Comment:
I figured adding lodash, or some other heavy-weight clone tool is unnecessary just for Node 16, since that will be EOL in [a matter of weeks](https://nodejs.org/en/blog/announcements/nodejs16-eol). Until then, this just means Node 16 mutates the original reference and doesn’t receive this bugfix.